src/osd/PrimaryLogPG.cc

```diff
@@ -6089,12 +6089,6 @@ int PrimaryLogPG::do_osd_ops(OpContext *ctx, vector<OSDOp>& ops)
       }
       break;
 
-    case CEPH_OSD_OP_STARTSYNC:
-      tracepoint(osd, do_osd_op_pre_startsync, soid.oid.name.c_str(), soid.snap.val);
-      t->nop(soid);
-      break;
-
-
```

Comment:
@liewegas Do i need to keep it.. the definition of `do_osd_op_pre_startsync` function?